pages/osx/osascript.md

```diff
@@ -0,0 +1,27 @@
+# osascript
+
+> Run AppleScript or JavaScript for Automation (JXA) from the command line.
+
+- Run an AppleScript command:
+
+`osascript -e '{{say "Hello world"}}'`
+
+- Run multiple AppleScript commands:
+
+`osascript -e '{{say "Hello"}}' -e '{{say "world"}}'`
+
+- Run an AppleScript file:
+
+`osascript {{path/to/apple.scpt}}`
```

Comment:
Well, it can either be `.scpt`, `.scptd` or `.applescript` (The former 2 are usually compiled, with `.scptd` being a 'script bundle' and the latter AppleScript written in plain text), but yeah it seems to choke unless one of those file extensions is used… So not sure what you want to do with that?